stack.yaml

```diff
@@ -41,6 +41,19 @@ packages:
 # (e.g., acme-missiles-0.3)
 extra-deps:
   - vendor/bson-generic
+  - fromhtml-0.1.0.0
+  - pandoc-2.2.1
```

Comment:
Do we need all these dependencies? Like Pandoc, etc.

Reply:
Sadly yes, without that it won't compile probably thanks to using lts-9.11 (with cabal.config) where are older versions, e.g., `pandoc == 1.19.2.4`...